build/WindowsAppSDK-Foundation-Nightly.yml

```diff
@@ -33,6 +33,30 @@ parameters:
   displayName: "Run Static Analysis (e.g., PREFast, APIScan)"
   type: boolean
   default: true
+- name: "BuildSampleApps"
+  displayName: "Run Build Sample Apps Stage"
+  type: boolean
+  default: true
+- name: "TestSampleAppsAfterBuildStage"
+  displayName: "Test launch of Sample apps separately in the TestSampleApps Stage"
+  type: boolean
+  default: true
+- name: "TestOnArm64"
+  displayName: "Enable running of sample tests on arm64 platform (Default: true)"
+  type: boolean
+  default: true
+- name: maxParallelForBuildSamplesCompatJob_x64
```

Comment:
I think the number of parameters are getting too overwhelming, considering moving maxParallelForBuildSamplesCompatJob_x64 and maxParallelForBuildSamplesCompatJob_arm64 to a variable set from the pipeline instead.